Automattic/crowdsignal-plugin
# Fix table layout

Fixes #95

#### Changes proposed in this Pull Request:

A new grid layout is used for the table, making it more width/mobile friendly.

#### Testing instructions:

Checkout and visit the Crowdsignal page. Check the titles, even truncated, will display full text when hovering (for a couple of seconds). Switch to some mobile view, see that the list is browsable.
